src/metadata_tool/core.clj

```diff
@@ -35,8 +35,7 @@
     "gen-bitergia-project-data"      tgn/gen-bitergia-project-data
     "gen-clabot-whitelist"           tgn/gen-clabot-whitelist
     "gen-catalogue-data"             tgn/gen-catalogue-data
-    "email-pmc-reports"              trp/email-pmc-reports
-  })
+    "email-pmc-reports"              trp/email-pmc-reports})
```

Comment:
Note: this issue is pervasive throughout the PR, but I haven't marked all instances.  Please find & revert all instances before resubmitting the PR for review.

Reply:
This is one of the unusual choices in this codebase. Almost no clojure code in the while closes brackets or parens on a newline. Tools know how to semantically treat "comment last entry of a sequence object" and push the closing bracket on the next line. I think this introduces visual noise without bringing the value it is supposed to bring.

Reply:
_Some_ tools know how to do this, but by no means all.  The Clojure coding style adopted by FINOS is intended to avoid creating incidental work for any developer, regardless of their tools of choice.

That said, the use of closing brackets on newlines is not consistent throughout the code base - it is primarily used for static data structures that change often during development and/or are regularly tested with various elements commented out; this form being a good example of that.